example/unicorn/templates/unicorn/js.html

```diff
@@ -54,4 +54,12 @@ <h2>Select2</h2>
             select2_datetime: {{ select2_datetime }}
         </div>
     </div>
+
+    <div>
+        <form wire:submit.prevent="upload_file">
```

Comment:
Is this meant to be `unicorn:submit`?

Reply:
Yes! This would need to be `unicorn:submit` once this is all working.